ui/src/main/resources/edu/wpi/grip/ui/codegeneration/java/operations/Find_Blobs.vm

```diff
@@ -0,0 +1,55 @@
+	/**
+	 * Detects groups of pixels in an image.
+	 * @param input The image on which to perform the find blobs.
+	 * @param minArea The minimum size of a blob that will be found
+	 * @param circularity The minimum and maximum circularity of blobs that will be found
+	 * @param darkBlobs The boolean that determines if light or dark blobs are found.
+	 * @param blobList The output where the MatOfKeyPoint is stored.
+	 */
+	private void $tMeth.name($step.name())(Mat input, double minArea, double[] circularity,
+		Boolean darkBlobs, MatOfKeyPoint blobList) {
+		FeatureDetector blobDet = FeatureDetector.create(FeatureDetector.SIMPLEBLOB);
+		try {
+			File tempFile = File.createTempFile("config", ".xml");
+
+			StringBuilder config = new StringBuilder();
+
+			config.append("<?xml version=\"1.0\"?>\n");
+			config.append("<opencv_storage>\n");
+			config.append("<thresholdStep>10.</thresholdStep>\n");
+			config.append("<minThreshold>50.</minThreshold>\n");
+			config.append("<maxThreshold>220.</maxThreshold>\n");
+			config.append("<minRepeatability>2</minRepeatability>\n");
+			config.append("<minDistBetweenBlobs>10.</minDistBetweenBlobs>\n");
+			config.append("<filterByColor>1</filterByColor>\n");
+			config.append("<blobColor>");
+			config.append((darkBlobs ? 0 : 255));
+			config.append("</blobColor>\n");
+			config.append("<filterByArea>1</filterByArea>\n");
+			config.append("<minArea>");
+			config.append(minArea);
+			config.append("</minArea>\n");
+			config.append("<maxArea>");
+			config.append(Integer.MAX_VALUE);
+			config.append("</maxArea>\n");
+			config.append("<filterByCircularity>1</filterByCircularity>\n");
+			config.append("<minCircularity>");
+			config.append(circularity[0]);
+			config.append("</minCircularity>\n");
+			config.append("<maxCircularity>");
+			config.append(circularity[1]);
+			config.append("</maxCircularity>\n");
+			config.append("<filterByInertia>0</filterByInertia>\n");
+			config.append("<filterByConvexity>0</filterByConvexity>\n");
+			config.append("</opencv_storage>\n");
```

Comment:
Is this really the best way to do this?
Could this be another file that gets pulled in as a resource? Then it could be actuall parseable xml